py/picca/delta_extraction/astronomical_objects/forest.py

```diff
@@ -343,11 +264,11 @@ def get_data(self):
             units += ["log Angstrom"]
             array_size = self.log_lambda.size
         elif Forest.wave_solution == "lin":
```

Comment:
I understand that log_lambda is the variable used now by the Forest class. Wouldn't it be possible then to store just log_lambda always, regardless of what the wave solution in reading was?

Reply:
Maybe wavelength itself is a little nicer as the unit can be put into the file explicitely (loglam is both unitless and could follow different log-definitions in principle), with putting Lamba here there would be a unique definition. Also at least for DESI purposes linear binning should become the norm, so this would be a future vs past choice where I'd opt for future...

Reply:
Yes, we can do it like this. In that case I think we should store `wave solution` (and maybe `delta_lambda` or `delta_log_lambda`) as it might be useful to know how were the files computed

Reply:
Just to add to what Michael was saying, `lin` is already the default choice. I'm happy to do it either way (always saving `log_lambda` or saving `lambda` depending on the choice for wavelength solution)

Reply:
I agree with Michael that `lambda` is a bit better than log-lambda. However, it would be great if we could still read older delta files that had `log_lambda`. And once you accept that you'll have to be able to read from both formats, it is not too bad if we also allow the writing to be in two formats. So I'm happy to leave the two options as it is now.

Reply:
Ah, sorry. Misread this. Agreed that saving in whatever format was used in the computation makes a lot of sense.